UI/zip-it/src/App.tsx

```diff
@@ -302,43 +325,63 @@ function App() {
       .then((result) => {
         console.log(result);
         toast({
-          title: "Account Created",
-          description: "Your listing has been successfully created.",
-          status: "success",
+          title: 'Listing Created',
+          description: 'Your listing has been successfully created.',
+          status: 'success',
           duration: 2000,
           isClosable: true,
-          position: "top",
+          position: 'top',
         });
         setNewListingVisible(false);
       })
 
       .catch((result) => {
-
         toast({
-          title: "Catch Error",
-          description: "Listing has encountered an error.",
-          status: "error",
+          title: 'Catch Error',
+          description: 'Listing has encountered an error.',
+          status: 'error',
           duration: 2000,
           isClosable: true,
-          position: "top",
+          position: 'top',
         });
 
-        console.log("Apollo/GraphQL failure - Zip-It");
-        console.log("check relevant query in queries.tsx");
+        console.log('Apollo/GraphQL failure - Zip-It');
+        console.log('check relevant query in queries.tsx');
         console.log(props);
         console.log(result);
 
         setNewListingDisabled(false);
       });
   };
 
+  //portal type selection logic
+  function UserAdminPortalDisplay(logInDetails) {
+    if (
+      authenticated?.userEmail ===
```

Comment:
I think this is an odd way to check for this, I understand it's early but the User Model has a build in Role number which you can grab and store as part of the logInDetails which can be used for this check on any account (ie. if role = 1, etc)

Reply:
Exactly, this was for testing purposes, RoleID = 1 is the proper function call for admin. I'll have to have a chat re: if there's an admin user with RoleID1 on the backend we can all log into and have suspended this PR until this is implemented.

Reply:
Testing complete, admin portal is now only accessed by RoleID via query.